hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -53,15 +53,14 @@ profile::buildmaster::cloud_agents:
           cpus: 4
           memory: 8
         - name: jnlp-maven-11
-          image: jenkinsciinfra/inbound-agent-maven@sha256:0ddce98de47930b7505418421b89cbbfa5378d1c8f9c01bee416ff8ffe1d0775
```

Comment:
I'm surprised to see the deletion of an image specification.  Is the image then selected purely by name?  Is that enough?